### PR TITLE
feat: add app icon

### PR DESCRIPTION
**Pull request by mhenriette**

### CODE REVIEW

File: contexts/AuthContext.tsx [hunk 1]
```diff
@@ -5,8 +5,8 @@ import React, {
   useContext,
   ReactNode,
 } from "react"
-import AsyncStorage from "@react-native-async-storage/async-storage"
 import * as SplashScreen from "expo-splash-screen"
+import AsyncStorage from "@react-native-async-storage/async-storage"
```
Comment (igorntwari) [inline contexts/AuthContext.tsx:9]: this change is unnecessary

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "expo-dev-client": "~4.0.28",
     "expo-font": "~12.0.9",
     "expo-linking": "~6.3.1",
-    "expo-splash-screen": "~0.27.5",
+    "expo-splash-screen": "~0.27.7",
```
Comment (igorntwari) [inline package.json:32]: why did you update it

File: contexts/AuthContext.tsx (hunk 1 above)
Comment (mhenriette) [reply contexts/AuthContext.tsx:9]: what do you mean, the AsyncStorage is being used

File: package.json (hunk 2 above)
Comment (mhenriette) [reply package.json:32]: I updated the splash screen for improvement and to keep up with the latest version.

File: contexts/AuthContext.tsx (hunk 1 above)
Comment (igorntwari) [reply contexts/AuthContext.tsx:9]: yes but it was already there you did not have to move it

**Review by igorntwari — APPROVED** [review]

**Review by serapieTuyishime — APPROVED** [review]

**Pull request merged**